docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -422,7 +422,7 @@ It is a vector of shared pointers to |FlowControllerDescriptor-api|, which has t
      - Type
      - Default Value
    * - |FlowControllerDescriptor::name-api|
-     - ``const char *``
+     - ``string``
```

Comment:
I think that the [Publish mode qos' flow controller name](https://github.com/eProsima/Fast-DDS-docs/blob/2178f2e41afb97f4e5c5ea3e003ed5858345dd79/docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst?plain=1#L739) should be also a `string` instead of a `const char *`

Reply:
Definitely